tests/share/models/factories.py

```diff
@@ -144,3 +144,38 @@ class TagFactory(ShareObjectFactory):
 
     class Meta:
         model = models.Tag
+
+
+class ThroughTagsFactory(ShareObjectFactory):
+    tag = factory.SubFactory(TagFactory)
+    creative_work = factory.SubFactory(AbstractCreativeWorkFactory)
+
+    class Meta:
+        model = models.ThroughTags
+
+
+class WorkIdentifierFactory(ShareObjectFactory):
```

Comment:
On purpose, they build different things. The one in normalize builds dictionaries. This one builds models.